Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -42,6 +42,8 @@ public enum FeatureFlag: String, CaseIterable {
 
     case cachePlayingEpisode
 
+    case playerFiltersFix
```

Comment:
I'd change that to `defaultPlayerFilterCallbackFix`. Adding a comment linking to the crash would also be good so it gives more context to someone looking at removing/changing that.